commons/src/main/java/org/infinispan/commons/jmx/JmxUtil.java

```diff
@@ -69,8 +71,9 @@ public static void registerMBean(Object mBeanInstance, ObjectName objectName, MB
          }
       } catch (InstanceAlreadyExistsException e) {
          //this might happen if multiple instances are trying to concurrently register same objectName
-         log.couldNotRegisterObjectName(objectName, e);
-         throw e;
+         throw CONTAINER.jmxMBeanAlreadyRegistered(objectName);
```

Comment:
I liked it better as it was before. This is not the proper place to handle domain conflicts. It should be done in AbstractJmxRegistration.

Also, the error message, mentioning cache manager name is too specific and not really needed. This is supposedly a generic utility method, used even by remote client.

Reply:
The way it was before it would wrap the `InstanceAlreadyExistsException` in a `CacheException` with no hint to the user about what went wrong, so I'm not a fan.